src/api/dexPriceEstimator/DexPriceEstimatorApi.ts

```diff
@@ -37,9 +38,16 @@ export interface Params {
 }
 
 function getDexPriceEstimatorUrl(networkId: number): string {
-  const networkName = getNetworkFromId(networkId).toLowerCase()
-  // TODO: use prod endpoint when available https://github.com/gnosis/dex-react/issues/869
-  return `https://price-estimate-${networkName}.dev.gnosisdev.com/api/v1/`
+  const basePath = 'api/v1/'
+
+  switch (networkId) {
+    case Network.Mainnet:
+      return `https://dex-price-estimator.gnosis.io/${basePath}`
+    case Network.Rinkeby:
+      return `https://price-estimate-rinkeby.dev.gnosisdev.com/${basePath}`
```

Comment:
```suggestion
      return `https://dex-price-estimator.rinkeby.gnosis.io/${basePath}`
```

as per @fleupold's [comment](https://github.com/gnosis/dex-react/issues/869#issuecomment-613873214)